vision_stack/camera/bottomcam/src/main.cpp

```diff
@@ -0,0 +1,79 @@
+#include "bottomcam/bottomcam.h"
+
+Camera camObj;
+
+void msgCallback(const std_msgs::String::ConstPtr& msg)
+{
+    int camMsg = atoi(msg->data.c_str());
+
+    if (camMsg == camObj.getCurrentCameraNo() && !camObj.isOpen())
+    {
+        if (camObj.returnCamera().open(camMsg))
+        {
+            std::cout << "Camera " << camMsg << " opened successfully.";
+            camObj.setCameraNo(camMsg);
+        }
+    }
+    else
+        if (camMsg != camObj.getCurrentCameraNo() && camObj.isOpen())
+        {
+            camObj.returnCamera().release();
+            std::cout << "Camera " << camObj.getCurrentCameraNo() << " closed successfully";
+            camObj.setCameraState(false);
+        }
+}
+
+int main(int argc, char** argv)
+{
+
+    camObj.inspect();
+
+    ros::init(argc, argv, "bottomcam");
+
+    ros::NodeHandle _nh;
+    image_transport::ImageTransport _it(_nh);
+    image_transport::Publisher _image_pub = _it.advertise(topics::CAMERA_BOTTOM_RAW_IMAGE, 1);
+    ros::Subscriber _sub = _nh.subscribe(topics::CAMERA_CAM_SWITCH, 1, msgCallback);
+
+    sensor_msgs::ImagePtr _publishImage;
+    cv_bridge::CvImage _image;
+    _image.encoding = "bgr8";
+
+    cv::VideoCapture _this_cam = camObj.returnCamera();
+    cv::VideoCapture cam;
+
+    if (argc >= 2)
+    {
+        int _cameraNo = atoi(argv[1]);
+        camObj.setCameraNo(_cameraNo);
+        _this_cam.open(_cameraNo);
```

Comment:
I think we should also handle the case when this function call fails. And then return an error message specifying this.